tests/unit/jobserver/test_rap_api.py

```diff
@@ -247,3 +248,37 @@ def test_bad_status_code(self, patch_api_call):
         with pytest.raises(RapAPIResponseError, match="gory") as exc:
             cancel(*self._fake_args)
         assert exc.value.body == fake_json
+
+
+class TestStatus:
+    """Tests of status.
+
+    This just returns the response and doesn't distinguish between non-200
+    error codes, so these are pretty simple."""
+
+    _fake_args = ["abcdefgh12345678"]
+
+    def test_success(self, patch_api_call):
+        """Test api_call is called with expected parameters and its body
+        returned when it returns a 200."""
+        fake_json = {"jobs": [{"identifier": "abcdefgh12345678"}]}
+        mock_api_call = patch_api_call(fake_json)
+        result = status(self._fake_args)
+
+        assert result == fake_json
+        mock_api_call.assert_called_once_with(
+            requests.post,
+            "rap/status/",
+            {
+                "rap_ids": self._fake_args,
+            },
+        )
+
+    def test_bad_status_code(self, patch_api_call):
```

Comment:
Could test what happens if the response body isn't json? (Again, probably not for this PR)

Reply:
yes will do while I'm doing